ansible.mk

```diff
@@ -0,0 +1,192 @@
+# kernel-style V=1 build verbosity
+ifeq ("$(origin V)", "command line")
+  BUILD_VERBOSE = $(V)
+endif
+
+ifeq ($(BUILD_VERBOSE),1)
+  Q =
+else
+  Q = @
+endif
+
+VERSION = $(shell git describe --dirty --tags --always)
+GIT_COMMIT = $(shell git rev-parse HEAD)
+K8S_VERSION = v1.18.2
+# TODO: change path to ansible repo
+REPO = github.com/operator-framework/operator-sdk
+PKGS = $(shell go list ./... | grep -v /vendor/)
+# TODO: change path to ansible repo
+TEST_PKGS = $(shell go list ./... | grep -v -E 'github.com/operator-framework/operator-sdk/test/')
+SOURCES = $(shell find . -name '*.go' -not -path "*/vendor/*")
+# GO_BUILD_ARGS should be set when running 'go build' or 'go install'.
+GO_BUILD_ARGS = \
+  -gcflags "all=-trimpath=$(shell go env GOPATH)" \
+  -asmflags "all=-trimpath=$(shell go env GOPATH)" \
+  -ldflags " \
+    -X '$(REPO)/version.GitVersion=$(VERSION)' \
+    -X '$(REPO)/version.GitCommit=$(GIT_COMMIT)' \
+    -X '$(REPO)/version.KubernetesVersion=$(K8S_VERSION)' \
+  " \
+
+
+DEFAULT_IMAGE = quay.io/operator-framework/ansible-operator
+IMAGE ?= $(DEFAULT_IMAGE)
+ARCHES:="amd64" "ppc64le" "s390x" "arm64"
+
+export CGO_ENABLED:=0
+.DEFAULT_GOAL:=help
+
+.PHONY: help
+help: ## Show this help screen
+	@echo 'Usage: make <OPTIONS> ... <TARGETS>'
+	@echo ''
+	@echo 'Available targets are:'
+	@echo ''
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##############################
+# Development                #
+##############################
+
+##@ Development
+
+.PHONY: all install
+
+all: format test build/ansible-operator ## Test and Build the ansible-operator
+
+install: ## Install the ansible-operator binary
+	$(Q)$(GOARGS) go install $(GO_BUILD_ARGS) ./cmd/ansible-operator
+
+# Code management.
+.PHONY: format tidy clean cli-doc lint
+
+format: ## Format the source code
+	$(Q)go fmt $(PKGS)
+
+tidy: ## Update dependencies
+	$(Q)go mod tidy -v
+
+clean: ## Clean up the build artifacts
+	$(Q)rm -rf build
+
+lint-dev:  ## Run golangci-lint with all checks enabled (development purpose only)
+	./hack/tests/check-lint.sh dev
+
+lint-fix: ## Run golangci-lint automatically fix (development purpose only)
+	./hack/tests/check-lint.sh fix
```

Comment:
IMO we will have just the lint check and it will not so longer setup with shell as it is done in sdk. 
and then we will have the file `.golangci.yml` to setup the lint as it is done in kubebuilder, cotroller-runtime and joes helm-operator as well. 

Also, the targets could be replaced as @joelanford made in his project https://github.com/joelanford/helm-operator/blob/master/Makefile#L42-L53 following up the kb Makefile layout to install its deps.

Reply:
Works for me